lib/watchbuild/runner.rb

```diff
@@ -59,18 +60,18 @@ def wait_for_build(start_time)
     end
 
     def notification(build, minutes)
-      require 'terminal-notifier'
-
       if build.nil?
         UI.message 'Application build is still processing'
         return
       end
 
       url = "https://appstoreconnect.apple.com/WebObjects/iTunesConnect.woa/ra/ng/app/#{@app.apple_id}/activity/ios/builds/#{build.train_version}/#{build.build_version}/details"
-      TerminalNotifier.notify('Build finished processing',
-                              title: build.app_name,
-                              subtitle: "#{build.train_version} (#{build.build_version})",
-                              execute: "open '#{url}'")
+
+      if !ENV['SLACK_URL'].empty?
```

Comment:
super nitty, but I'd invert this, it's usually easier to read things where the statement should be truthy